packages/core/src/tracing/cls.js

```diff
@@ -349,17 +349,19 @@ function setCurrentSpan(span) {
 
 /**
  * Get the currently active span.
+ * @param {boolean} [fallbackToSharedContext=false]
  * @returns {InstanaSpan}
  */
-function getCurrentSpan() {
-  return ns.get(currentSpanKey);
+function getCurrentSpan(fallbackToSharedContext = false) {
```

Comment:
@basti1302 , again, about the new optional param from https://github.com/instana/nodejs/pull/416#discussion_r760925276

Reply:
There is one more in the same place, doing the same thing, so I won't tag you there again, as they are pretty much the same thing, in the same file